Biblio.bib

```diff
@@ -99,20 +99,24 @@ @InProceedings{laurie2004proof
 }
 
 @Misc{vishnumurthy03karma:a,
+    url = "https://www.cs.cornell.edu/people/egs/papers/karma.pdf",
     author = "Vivek Vishnumurthy and Sangeeth Chandrakumar and Emin Gün Sirer",
     title = "{KARMA}: A Secure Economic Framework for Peer-to-Peer Resource Sharing",
     year = "2003"
 }
 
 @InProceedings{dwork92pricingvia,
+    url = "http://www.wisdom.weizmann.ac.il/~naor/PAPERS/pvp.pdf",
     author = "Cynthia Dwork and Moni Naor",
     title = {Pricing via processing or combatting junk mail},
     booktitle = {In 12th Annual International Cryptology Conference},
     year = {1992},
-    pages = {139--147}
+    pages = {139--147},
 }
 
 @Article{nakamoto2008bitcoin,
+	url = {{http://www.bitcoin.org/bitcoin.pdf}},
+	note = "The URL is recommended by the author \href{https://www.mail-archive.com/cryptography@metzdowd.com/msg09959.html}{here}.",
```

Comment:
I think this should be just a comment in the `.bib` file.

Reply:
Done.